src/pybind/mgr/dashboard/services/cephfs.py

```diff
@@ -220,7 +220,9 @@ def set_quotas(self, path, max_bytes=None, max_files=None):
         :param max_files: The file limit.
         :type max_files: int | None
         """
-        self.cfs.setxattr(path, 'ceph.quota.max_bytes',
-                          str(max_bytes if max_bytes else 0).encode(), 0)
-        self.cfs.setxattr(path, 'ceph.quota.max_files',
-                          str(max_files if max_files else 0).encode(), 0)
+        if max_bytes is not None:
+            self.cfs.setxattr(path, 'ceph.quota.max_bytes',
+                              str(max_bytes).encode(), 0)
+        if max_files is not None:
+            self.cfs.setxattr(path, 'ceph.quota.max_files',
+                              str(max_files).encode(), 0)
```

Comment:
We only set `ceph.quota.max_bytes` for a subvolume create and resize.
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/volumes/fs/subvolume.py#L85
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/volumes/fs/subvolume.py#L177

Reply:
As per our conversation, both quotas are working and only those values can be set, therefore I will resolve the conversation.